pkg/controller/persistentvolume/types.go

```diff
@@ -26,6 +26,14 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
```

Comment:
Please don't let what I say (about names in particular) stop you from doing sensible things.  That said, I think "class" is a better name - it's less about quality and more about particular shape.